var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -94,6 +99,7 @@ class Elemental(CMakePackage):
     depends_on('gmp', when='+mpfr')
     depends_on('mpc', when='+mpfr')
     depends_on('mpfr', when='+mpfr')
+    depends_on('cmake -openssl -ncurses')
```

Comment:
Why `-openssl` and `-ncurses`?

Reply:
Because those add a number of dependencies and build time that are obxonious to wait for. Given that this cmak package does not need those features I don’t think that all cmake packages should have to inherit those variants being turned on. Also one of the dependent packages doesn’t compile with the XL compiler.

Reply:
> Because those add a number of dependencies and build time that are obxonious to wait for

you just need to add those to your `packages.yaml`. The package itself is not a place to add those constraints unless it is **known not to work otherwise**. So please remove this line altogether as `CMake` package already depends on `cmake`.

Reply:
> Also one of the dependent packages doesn’t compile with the XL compiler.

feel free to open an issue for this, perhaps someone with `xl` compiler can have a look.

Reply:
Is there a way to depend on any cmake?